c172p-fdm-dany93.xml

```diff
@@ -939,13 +950,23 @@
                     <value>0.0147</value>
                 </product>
             </function>
-            <function name="aero/coefficient/Clbk">
-                <description>Roll_moment_due_to_one_broken wing</description>
+            <function name="aero/coefficient/Cllwdmg">
```

Comment:
Yes, I must have been looking at an older version of the file it seems :frowning:  :sweat_smile: